### PR TITLE
Update Banner with Google Map Integration

### PR DESCRIPTION
**Pull request by hassankarrach**
This pull request updates the banner by incorporating a Google Map element and introduces a new file, Data.js, to store location information for villages (for marker placement on the map).

Changes:

Added Google Map integration to the banner.
Introduced Data.js to hold village location data for map markers.

Please review and merge.

image

### CODE REVIEW

**Review by soufyanelf — CHANGES_REQUESTED** [review]

**Comment by soufyanelf** [comment]
@hassankarrach Are you working on the map integration? if not I'll jump into it sometime next week

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -2,60 +2,134 @@
 import Link from "next/link";
 import Image from "next/image";
 import styles from "./header.module.css";
+//MaterialUI
```
[the remark this answers is not in the document]
Comment (hassankarrach) [reply src/components/Header/index.jsx:5]: I'm a bit unsure about whether toggling between the map and the list is the perfect choice.

[the remark this answers is not in the document]
Comment (soufyanelf) [reply src/components/Header/index.jsx:5]: If you want ui/ux feedback and maybe a mockup please highlight this discussion in the mosa3ada-ui-ux channel in discord https://discord.com/invite/pjyW6Zje

**Comment by soufyanelf** [comment]
Im going to close this PR. we are not going to use Google Maps we will use Mapbox instead.

**Pull request closed without merging**